domotifications-android/Domotifications/app/src/main/java/com/lasalle/domotifications/BaseDeDonnees.java

```diff
@@ -614,4 +614,28 @@ public void insererModule(int     idModule,
             Log.e(TAG, "Erreur d'insertion du module");
         }
     }
+
+    public void supprimerModule(int     idModule,
+                                int     idTypesModules,
+                                String  nom,
+                                boolean actif,
+                                String  couleur)
+    {
+        Log.d(TAG,
+              "supprimerModule() idModule = " + idModule + " idTypesModules = " + idTypesModules +
+                " nom = " + nom + " actif = " + actif + " couleur = " + couleur);
+        try
+        {
+            String requete =
+              "DELETE FROM modules WHERE id = " + idModule +
```

Comment:
Non, la condition est limitée à la clé qui est `WHERE id = " + idModule + " AND idTypesModules = " + idTypesModules`